utils/display-field.h

```diff
@@ -0,0 +1,74 @@
+#ifndef UFTRACE_DISPLAY_FIELD_H
+#define UFTRACE_DISPLAY_FIELD_H
+
+#include "utils/fstack.h"
+#include "utils/list.h"
+
+static LIST_HEAD(output_fields);
+
+/* data for field display */
+struct field_data {
+	struct ftrace_task_handle *task;
+	struct fstack *fstack;
+	void *arg;
+};
+
+enum display_field_id {
+	REPLAY_F_NONE           = -1,
+	REPLAY_F_DURATION,
+	REPLAY_F_TID,
+	REPLAY_F_ADDR,
+	REPLAY_F_TIMESTAMP,
+	REPLAY_F_TIMEDELTA,
+	REPLAY_F_ELAPSED,
+};
+
+struct display_field {
+	struct list_head list;
+	enum display_field_id id;
+	const char *name;
+	const char *header;
+	int length;
+	bool used;
+	void (*print)(struct field_data *fd);
+};
+
+static int print_field_data(struct field_data *fd)
+{
```

Comment:
Hum.. I thought it seems good that a header file contains data structure, variables,
function headers, inline function definitions and etc.

But this header file includes function definitions.
Is it OK on coding convention of uftrace ?

Reply:
I moved it into a new file `utils/field.c`. Thanks!